bitnami/nginx/values.yaml

```
@@ -185,6 +185,13 @@ service:
     http: ""
     https: ""
 
+  ## Target port reference value for the Loadbalancer service types can be specified explicitly.
+  ## Listeners for the Loadbalancer can be custom mapped to the http or https service.
+  ## Example: Mapping the https listener to targetPort http [http: https]
+  targetPort:
+    http: ""
+    https: ""
```

The default values should be `http` and `https`:
```suggestion
    http: http
    https: https
```